ui/src/services/version/RouterVersion.js

```diff
@@ -0,0 +1,92 @@
+import yaml from "js-yaml";
+import objectAssignDeep from "object-assign-deep";
+
+export class RouterVersion {
+  static fromJson(json) {
+    return objectAssignDeep(new RouterVersion(), json);
+  }
+
+  toPrettyYaml() {
+    const pretty = {
+      version: this.version,
+      router: {
+        image: this.image,
+        timeout: this.timeout,
+        routes: this.routes.map(route => ({
+          id: route.id,
+          endpoint: route.endpoint,
+          timeout: route.timeout,
+          is_default: route.id === this.default_route_id
+        })),
+        resource_request: this.resource_request
+      },
+      experiment_engine: {
+        type: this.experiment_engine.type,
+        ...(this.experiment_engine.type !== "nop"
+          ? {
+              config: {
+                client: {
+                  id: this.experiment_engine.config.client.username,
+                  passkey: "[hidden]"
```

Comment:
If the passkey changed between versions, do you think it would be of interest to the users? If the passkey is set, we can may be compute another hash over the already encrypted string and use it for compare?

Reply:
I was thinking about this, but I believe that this hashed string makes no sense to user at all. 
Instead, there should something simpler, such as just `[hidden]/[changed]` or similar.

> may be compute another hash over the already encrypted string
curious, why do we need to compute another hash?

Reply:
>why do we need to compute another hash?

I thought you had concerns with displaying the encrypted string as is in case it was long or insecure (not that the user can't get hold of this string otherwise).

Then, should we just call the key "encrypted_passkey" instead of "passkey" and put down the actual contents so it can still be compared?

Reply:
Okay, done